dbos/core.py

```diff
@@ -42,3 +80,273 @@ def get_status(self) -> WorkflowStatus:
         if stat is None:
             raise DBOSNonExistentWorkflowError(self.workflow_uuid)
         return stat
+
+
+def _init_workflow(
```

Comment:
We used the name starting with an underscore to indicate a "private" method within a class. If we refactor them out, we don't need the underscore as a prefix.

Reply:
I was sort of thinking of it as an extension of the class, and it makes grepping for the private internal vs public one a lot easier... so I thought it was nice to leave the '_' on there.  Private to the module...

Reply:
Sure it's not a big deal.